docs/manualGrading/index.md

```diff
@@ -94,10 +94,61 @@ The user interface for manual grading is built to encourage a workflow where all
+
+In the description, explanation, and grader note, it is possible to use mustache patterns based on the parameters, correct answers and submitted answers for a submission. This will cause the description of the item provided to graders and students to match the question variant assigned to the student, as shown in the screenshot below.
+
+![Rubric items in settings](rubric_items_setting.png)
+
+This will result in a grading panel containing a description with the value of the parameter replaced with the appropriate value for the student's variant.
+
+![Rubric items in grading panel](grading_panel_with_rubrics.png)
+
+Student's feedback is also replaced in a similar manner.
+
+![Rubric items in student feedback](student_feedback_with_rubrics.png)
+
+As grading progresses, new scenarios may be found by graders that require changes in the rubric, such as new types of penalties or credits or specific kinds of feedback. The rubric may be changed as required, even after grading is already underway. These changes may affect existing submissions:
+
+- Adding new rubric items does not affect the score of existing submissions;
+- Changing the points assigned to individual rubric items will cause any existing submission gradings that use the rubric item to be recalculated with the new points assigned to the items;
+- Changing the description or explanation of individual rubric items will update the feedback provided to students, but not the score;
+- Deleting rubric items will cause existing gradings that use the rubric item to be recalculated without the item;
+- Changing the minimum score or maximum extra credit will cause all existing rubric gradings to be recalculated.
+
+When updating rubric settings, a flag at the bottom of the settings dialog box gives graders the option of tagging all submissions for manual grading. This option may be used in cases where the changes are significant enough that they warrant a review of all previously graded questions to ensure they take the new rubric into account. This is encouraged in cases where rubric items applied to multiple rubric gradings are deleted.
+
+To grade a submission, a grader must select the items that apply to a particular submission and press Submit. Additionally, each rubric item (or the first 10 items, in long rubrics) is assigned a keyboard shortcut corresponding to a digit; pressing that digit on the keyboard causes that item to be selected (or cleared, if it is selected).
+
+Graders have the flexibility to apply a grading adjustment (positive or negative) on top of the rubric item. This may be used where instructors need to make judgement calls for unconventional submissions that don't apply to specific rubric items. This option is discouraged in cases where the issue is found in multiple submissions, as additional rubric items provide more flexibility in providing feedback and adjusting grades.
```

Comment:
Perhaps a screenshot of this might be useful ? this option is actually not that discoverable. Two options:

<img width="542" alt="Screenshot 2024-03-02 at 12 36 01 AM" src="https://github.com/PrairieLearn/PrairieLearn/assets/2507459/900170f4-f955-4c58-b2cb-db8571af0deb">

<img width="548" alt="Screenshot 2024-03-02 at 12 36 24 AM" src="https://github.com/PrairieLearn/PrairieLearn/assets/2507459/c431ad48-478b-459f-97c5-12fd7cee0789">

Reply:
I think adding a screenshot here may encourage this feature more than it's supposed to. I want to document that it exists, but not encourage its use extensively.